builder/builder.go

```diff
@@ -1737,13 +1741,47 @@ func (b *Builder) BuildImage(format string, template string) error {
 		format = b.State.Mix.Format
 	}
 
-	// If the user did not pass in a template, default to release-image-config.json
+	// swupd (client) called by itser will need a temporary directory to act as its stage dir.
+	wd, err := os.Getwd()
```

Comment:
I don't like this approach. Don't be looking in the current working directory, look in the mixer working directory as defined in the conf. Users can call mixer from anywhere and define a different mixer working directory. Case in point is the Clear Linux devops process.